src/components/SingleList.jsx

```diff
@@ -1,12 +1,18 @@
 import './SingleList.css';
+import { useStateWithStorage } from '../utils';
+
+export function SingleList({ name, index, path }) {
+	const [listPath, setListPath] = useStateWithStorage(
+		'tcl-shopping-list-path',
+		null,
+	);
 
-export function SingleList({ name, path, setListPath }) {
 	function handleClick() {
 		setListPath(path);
 	}
 
 	return (
-		<li className="SingleList">
+		<li className="SingleList" key={index}>
```

Comment:
Does the li element need a key prop if there is already a key prop on the SingleList component rendered in Home.jsx?